docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1769,134 +2016,16 @@ spring:
-
-Take into account that this filter to configure local response cache per route only will be available if the local response global cache is enabled.
-
-It accepts the first parameter to override the maximum size of the cache to evict entries for this route, it takes size format in KB, MB and GB; and a second parameter to override the time to expire a cache entry expressed in s for seconds, m for minutes and h for hours.
-
-The following listing shows how to add local response cache `GatewayFilter`:
-
-====
-[source,java]
-----
-@Bean
-public RouteLocator routes(RouteLocatorBuilder builder) {
-    return builder.routes()
-        .route("rewrite_response_upper", r -> r.host("*.rewriteresponseupper.org")
-            .filters(f -> f.prefixPath("/httpbin")
-        		.localResponseCache(Duration.ofMinutes(30), "500MB")
-            ).uri(uri))
-        .build();
-}
-----
-
-or this
-
-.application.yaml
-[source,yaml]
-----
-spring:
-  cloud:
-    gateway:
-      routes:
-      - id: resource
-        uri: http://localhost:9000
-        predicates:
-        - Path=/resource
-        filters:
-        - LocalResponseCache=30m,500MB
-----
-
-NOTE:  This filter also implements the automatic calculation of the max-age value in the HTTP Cache-Control header.
-Only if "max-age" is present on the original response the value will be rewritten with the number of seconds set in the timeToLive configuration parameter; and in consecutive calls this value will be recalculated with the number of seconds left until the response expires.
-====
-
-=== Modify a Response Body `GatewayFilter` Factory
-
-You can use the `ModifyResponseBody` filter to modify the response body before it is sent back to the client.
-
-NOTE: This filter can be configured only by using the Java DSL.
-
-The following listing shows how to modify a response body `GatewayFilter`:
-
-====
-[source,java]
-----
-@Bean
-public RouteLocator routes(RouteLocatorBuilder builder) {
-    return builder.routes()
-        .route("rewrite_response_upper", r -> r.host("*.rewriteresponseupper.org")
-            .filters(f -> f.prefixPath("/httpbin")
-        		.modifyResponseBody(String.class, String.class,
-        		    (exchange, s) -> Mono.just(s.toUpperCase()))).uri(uri))
-        .build();
-}
-----
-
-NOTE:  if the response has no body, the `RewriteFilter` will be passed `null`. `Mono.empty()` should be returned to assign a missing body in the response.
-====
 
-=== Token Relay `GatewayFilter` Factory
+=== The `TokenRelay` `GatewayFilter` Factory
 
 A Token Relay is where an OAuth2 consumer acts as a Client and
 forwards the incoming token to outgoing resource requests. The
 consumer can be a pure Client (like an SSO application) or a Resource
 Server.
 
 Spring Cloud Gateway can forward OAuth2 access tokens downstream to the services
-it is proxying. To add this functionlity to gateway you need to add the
+it is proxying. To add this functionality to gateway you need to add the
```

Comment:
Change to:

it is proxying. To add this functionality to the gateway, you need to add the

(I know you didn't break it. Thanks for fixing it anyway.)